src/print/index.ts

```diff
@@ -135,6 +138,19 @@ export function print(path: FastPath, options: ParserOptions, print: PrintFn): D
         case 'Head':
         case 'Title': {
             const isEmpty = node.children.every(child => isEmptyNode(child));
+
+            if (ignoreNext) {
```

Comment:
Are there other node types where we'd want to be respecting this comment? This is only run for `Element`, `InlineComponent`, `Slot`, `Window`, `Head`, and `Title` - and none of the other node types will reset the `ignoreNext` flag either.

Reply:
That's a good point. I don't know what my original reason for putting it here was. I've moved the check outside of the switch block. Should handle any case where a comment is valid.